packages/account/pages/login.vue

```diff
@@ -57,21 +64,19 @@ export default {
 		}
 	},
 	methods: {
-		submit() {
+		async submit() {
 			this.hasError = false
```

Comment:
Do you really need a `hasError` state for this when you already have `errorMessage` ? You can just check if the `errorMessage` type for the same effect.